### PR TITLE
asset metadata and mesuredat mapping correction

### PR DESCRIPTION
**Pull request by TisserantG**
## What does this PR do ?
 - asset metadata (and other information) copy in measures

### Other changes
- mesuredat mapping  chganged from double to date.

### CODE REVIEW

File: lib/core-classes/MeasureService.ts [hunk 1]
```diff
@@ -327,13 +351,14 @@ export class MeasureService {
...
-
+        const assetInfos = this.getAssetInfos(asset);
         validMeasures.push({
+          asset: assetInfos,
           assetMeasureName,
           deviceMeasureName: null,
           measuredAt: measurement.measuredAt || Date.now(),
           origin: {
-            assetId,
+            //assetId, //TODO : remettre et corriger
```
Comment (Aschen) [inline lib/core-classes/MeasureService.ts:361]: Comment should be in english

File: lib/mappings/measuresMappings.ts [hunk 2]
```diff
@@ -43,6 +45,22 @@ export const measuresMappings = {
       properties: {}
     },
 
+    // Asset linked to the device when the measure was made
+    //assetId: { type: 'keyword' },
```
Comment (Aschen) [inline lib/mappings/measuresMappings.ts:49]: Commented code should be removed

File: lib/types/measures/MeasureContent.ts [hunk 3]
```diff
@@ -56,6 +57,20 @@ export interface MeasureContent<TMeasurementValues extends JSONObject = JSONObje
    */
   unit: MeasureUnit;
 
+  /**
+   * Asset ID linked to the device when the measure was made
```
Comment (Aschen) [inline lib/types/measures/MeasureContent.ts:61]:
```suggestion
   * Asset linked to the device when the measure was made
```

**Review by Aschen — CHANGES_REQUESTED** [review]

File: features/AssetController.feature [hunk 4]
```diff
@@ -80,10 +80,10 @@ Feature: DeviceManager asset controller
       | size     | 3                          |
     Then I should receive a "measures" array of objects matching:
     # there is 6 measures with the 3 from fixtures
-      | _source.origin.assetId    | _source.origin.id                        |
-      | "container-FRIDGE-linked" | "DummyMultiTemp-attached_ayse_linked_1"  |
-      | "container-FRIDGE-linked" | "DummyMultiTemp-attached_ayse_linked_1"  |
-      | "container-FRIDGE-linked" | "DummyMultiTemp-attached_ayse_linked_1"  |
+      | _source.asset._id         | _source.origin.id                       |
```
Comment (Aschen) [inline features/AssetController.feature:83]: You should also check that the property of the `_source` object are also present

**Review by Aschen — CHANGES_REQUESTED** [review]

File: lib/core-classes/MeasureService.ts (hunk 1 above)
Comment (TisserantG) [reply lib/core-classes/MeasureService.ts]: done (removed)

File: lib/mappings/measuresMappings.ts (hunk 2 above)
Comment (TisserantG) [reply lib/mappings/measuresMappings.ts]: done

File: lib/types/measures/MeasureContent.ts (hunk 3 above)
Comment (TisserantG) [reply lib/types/measures/MeasureContent.ts]: done

File: features/AssetController.feature (hunk 4 above)
Comment (TisserantG) [reply features/AssetController.feature]: done

**Review by Aschen — APPROVED** [review]

**Pull request merged**